### ISSUE TITLE
Mistakes in Cycle ORM installation

### ISSUE THREAD

**Issue by Remo** [cycle/docs#103]
I tried to get Cycle ORM 2 running, but it seems like there are some things missing in the documentation. I've started here:

https://cycle-orm.dev/docs/intro-install/2.x/en

I then read this part: https://cycle-orm.dev/docs/intro-install/2.x/en#configuration-manually-configure-mapping (direct links to anchor don't seem to work, but there's an ID in the code). However, I don't want to skip the annotation part as I don't want to manually define my schema so I skip this part.

Next I therefore try to define my entity: https://cycle-orm.dev/docs/intro-install/2.x/en#define-entity. This works fine, the class loader also manages to find my class, however, in "Define Entity" we are creating the entity in the `src` folder and then scan `__DIR__`. If I try that, it will fail because that will try to load some classes in the `vendor` directory which can't be loaded because I don't have Illuminate/Collection installed. I replaced that with the `src` folder and managed to get it working.

I then tried to register my orm schema with this `$orm = $orm->with(schema: new \Cycle\ORM\Schema($schema));`, but here `$orm` isn't defined because I skipped the part where we manually register the schema. I also tried to keep it, but then I have a bunch of collisions with `Schema`..

I read the whole thing twice, but I don't see how the documentation here should work.

**Comment by butschster** [comment]
Hi @Remo. Thank you for detailed problem description! We appreciate it! Could you provide more information about Schema collisions?

**Comment by Remo** [comment]
Yes, that happened because I've had these two lines in my code as soon as I tried to use both approaches at the same time

```
use Cycle\ORM\Schema;
use Cycle\Schema;
```

(I know that I can alias that, but that shouldn't be necessary when I just install something)

There's also another mistake in the constructor of `User`. The constructor has two parameters, but it is initialized with this `$u = new \Example\User();`
